code/types.ts

```diff
@@ -0,0 +1,26 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult, Context } from "aws-lambda";
+
+export type ApiGatewayProxyHandler = (event: APIGatewayProxyEvent, context: Context) => Promise<APIGatewayProxyResult>
+
+export enum HttpStatus {
+    OK = 200,
+    NoContent = 204,
+    BadRequest = 400,
+    NotFound = 404,
+    ServerError = 500,
+}
+
+export interface MethodHandlerMap {
```

Comment:
I personally prefer using types and not interface (they are very similar but types allow you more flexibility when combining different definitions together).

Reply:
Thanks for the tip 👍